var/spack/repos/builtin/packages/hiptt/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Hiptt(MakefilePackage, ROCmPackage):
+    """hipTT - Fast GPU Tensor Transpose for NVIDIA and AMD GPU."""
+
+    homepage = "https://github.com/DmitryLyakh/hipTT"
+    url = "https://github.com/DmitryLyakh/hipTT"
+    git = "https://github.com/DmitryLyakh/hipTT.git"
+    tags = ["ecp", "radiuss"]
+
+    maintainers("bvanessen")
+
+    version("master", branch="master")
+
+    patch("bugfix_make.patch")
```

Comment:
Do you think we can get this patch upstreamed quickly? The recipe LGTM, but patching a moving target (the `master` branch) seems bound to cause failures in the future.

Reply:
@alalazo In principle I agree, and we can work to get this pushed upstream.  However, the repo is not in a state of high transition, so I don't think that this will be a problem.